api/services/containers/v1/containers.proto

```diff
@@ -114,8 +114,16 @@ message Container {
 	// to select a specific map key.
 	map<string, google.protobuf.Any> extensions = 10;
 
-	// Sandbox ID this container belongs to.
-	string sandbox = 11;
+	//  Sandboxer specifies the sandboxer name used for container sandbox
+	string sandboxer = 11;
+
+	// SandboxKey specifies the sandbox key to use for the container
+	// Before starting a task from this container, a caller should
+	// call AppendContainer to append container resource to the sandbox
+	// that the container to be launched in.
+	//
+	// This field is set to empty when the no sandbox specified.
+	string sandbox_key = 12;
```

Comment:
Execuse me I don't get if you mean the example of sandbox_key or the implemetation of our sandboxer?
If you mean the sandbox_key, it is actually a sandbox id created, like the field snapshot_key, which means the snapshot id for the container.

If you mean a implemetation of the sandboxer, Actually I add the a embeded "Sandboxer" based on runc. which I name it linux-sandbox, it can be run if you pull my branch in this PR and compile the containerd.

> To make this containerd running, we has to prepare a "pause" executable file, move it in `/bin` folder as it is the default location for runc sandboxer to find the pause executable. we do this to remove the dependency of Snapshotter for Sandboxer, and we don't need a pause image to start a sandbox anymore. then we did some modification in the config file of containerd
> 
> ```
> [plugins.cri.containerd.runtimes.runc]
>   runtime_type="io.containerd.runc.v2"
>   sandboxer="linux-sandbox"
> ```